feat(ReleaseNote): Add SAP connector section (#711)
bonitasoft/bonita-doc: md/release-notes.md

The version of the _org.twitter4j:twitter4j-core_ dependency has been updated from _4.0.2_ to _4.0.7_

#### SAP connector

> The SAP connector in the current version does not work with Java 11. No update of it are planned at the moment.

Reviewer: ```suggestion
The SAP connector in the current version does not work with Java 11. No update of this implementation is planned at the moment.
```
